### ISSUE TITLE
Double recording of mp4 files on android devices when running automation tests

### ISSUE THREAD

**Issue by dhreben** [zebrunner/mcloud-agent#131]
Steps:
1.Run automation tests on android device
2.Open. `docker exec -ti appium bash`
3.  `ps -ef`

Actual result:  Double recording of mp4 files on android devices when running automation tests

Logs:

```
root      1386    93  0 04:47 pts/0    00:00:00 /usr/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/linux/chromedriver_linux64_v73.0.3683.68 --url-base=wd/hub --port=8000 --adb-port=5037 --verbose
root      1920    93  0 05:02 pts/0    00:00:00 /bin/sh -c /opt/start-capture-artifacts.sh a510fc06-040e-4a1d-a2a1-876cccc3df33 > /tmp/video.log 2>&1
root      1921  1920  0 05:02 pts/0    00:00:00 /bin/bash /opt/start-capture-artifacts.sh a510fc06-040e-4a1d-a2a1-876cccc3df33
root      1950    93  0 05:02 pts/0    00:00:01 /root/platform-tools/adb -P 5037 -s FA74SBN02369 logcat -v threadtime
root      1976    93  0 05:02 pts/0    00:00:00 /root/platform-tools/adb -P 5037 -s FA74SBN02369 shell am instrument -w -e disableAnalytics true io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner
root      1984    93  0 05:02 pts/0    00:00:00 /bin/sh -c /opt/start-capture-artifacts.sh a510fc06-040e-4a1d-a2a1-876cccc3df33 >> /tmp/video.log 2>&1
root      1985  1984  0 05:02 pts/0    00:00:00 /bin/bash /opt/start-capture-artifacts.sh a510fc06-040e-4a1d-a2a1-876cccc3df33
root      2029    93  0 05:03 pts/0    00:00:00 /usr/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/linux/chromedriver_linux64_v73.0.3683.68 --url-base=wd/hub --port=8001 --adb-port=5037 --verbose
root      2208  1921  0 05:08 pts/0    00:00:00 adb shell screenrecord --verbose --bit-rate 2000000 /sdcard/a510fc06-040e-4a1d-a2a1-876cccc3df33_2.mp4
root      2210  1985  0 05:08 pts/0    00:00:00 adb shell screenrecord --verbose --bit-rate 2000000 /sdcard/a510fc06-040e-4a1d-a2a1-876cccc3df33_2.mp4
```

**Comment by vdelendik** [comment]
fixed by using appium:1.4.5

**Comment by dhreben** [comment]
Fixed
```
root        95    64  0 05:15 pts/0    00:00:33 node /usr/bin/appium --log-no-colors --log-timestamp --log /var/log/appium.log --nodeconfig /root/nodeconfig.json --default-capabilities /root/defaultcapabilities.json --relaxed-security --allow-insecure chromedriver_autodownload
root      6532    95  0 08:48 pts/0    00:00:00 /root/platform-tools/adb -P 5037 -s NE1G...2 logcat -v threadtime
root      6617    95  0 08:48 pts/0    00:00:00 /root/platform-tools/adb -P 5037 -s NE1G....2 shell am instrument -w -e disableAnalytics true io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner
root      6619     0  0 08:48 pts/1    00:00:00 bash
root      6643    95  0 08:48 pts/0    00:00:00 /bin/sh -c /opt/start-capture-artifacts.sh 9bc42a2a-db5b-4aac-b28e-d7bc39a92fc0 >> /tmp/video.log 2>&1
root      6644  6643  0 08:48 pts/0    00:00:00 /bin/bash /opt/start-capture-artifacts.sh 9bc42a2a-db5b-4aac-b28e-d7bc39a92fc0
root      6648  6644  0 08:48 pts/0    00:00:00 adb shell screenrecord --verbose --bit-rate 2000000 /sdcard/9bc42a2a-db5b-4aac-b28e-d7bc39a92fc0_0.mp4

```
